sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/lang/util/TreeUtil.kt

```diff
@@ -300,7 +300,7 @@ private fun ArrayList<SqlCreateTableStmt>.buildGraph(): Graph<SqlCreateTableStmt
   this.forEach { table ->
     graph.addVertex(table)
     table.columnDefList.forEach { column ->
-      column.columnConstraintList.mapNotNull { it.foreignKeyClause?.foreignTable }.forEach { fk ->
+      (column.columnConstraintList.mapNotNull { it.foreignKeyClause?.foreignTable }.asSequence() + table.tableConstraintList.mapNotNull { it.foreignKeyClause?.foreignTable }).forEach { fk ->
```

Comment:
Why the asSequence? There should be a plus overload for two lists, right?

Reply:
Yes, there is. The `asSequence()` is not required, but we can avoid creating an unnecessary intermediate list here.

Reply:
That's a mostly pointless optimization here. There's already two intermediate lists created from the two mapNotNull calls. If you wanted to avoid intermediate lists you need to make both sequences before the map operations.

Reply:
Good point. I'll just get rid of it.